### ISSUE TITLE
[Designer]Bug 31585214: [Usable - Adaptive Cards - Activity Update.Json]: Narrator does not read the selected date under the calendar window as focus lands by using Tab key.

### ISSUE THREAD

**Issue by aaagrawal** [microsoft/AdaptiveCards#7636]
### Problem Description

[Bug 31585214](https://microsoft.visualstudio.com/OS/_workitems/edit/31585214): [Usable - Adaptive Cards - Activity Update.Json]: Narrator does not read the selected date under the calendar window as focus lands by using Tab key.

User Experience:

Narrator users will be impacted if narrator will not read the selected date information first time.

Test Environment:

Application: Adaptive Cards

OS version: 2004 (OS Build 21301.1000)

Application Version: 2101.28.1.0

Pre-Requisite: Launch the Adaptive Card application.

Repro steps:

1. Launch the Adaptive Card application and Enable Narrator.

2. Adaptive Card UWP Visualizer screen starts appearing. 

3. Navigate to ActivityUpdate.Json button and activate it.

4. Publish Adaptive Card Schema Card will be appeared on right sight of the pane.

5. Navigate to Set due date button by using Tab key and activate it.

6. Navigate to Calendar date picker field by using Tab key and activate it.

7. Navigate to Selected date by using Tab key and verify the issue.

Actual Result:

Narrator does not read the selected date under the calendar window as focus lands by using Tab key. Narrator reads the date information when user navigate the dates by using left and right arrow keys. 

Observation:
Same issue is repro'ing with inputs.json and inputswithvalidation.json adaptive card.

Note:
Working fine with NVDA Screen Reader.

Expected result:

Narrator should announce the the selected date information while navigating by using Tab key. Narrator should announce as '3 Selected column header Wednesday 4 column of 7'. 

### Card JSON

```JSON
Designer Bug. Repro on UWP Adaptive Card visualizer
```
